### DISCUSSION TITLE
Do the `audit` and `forecast` commands support personal GitHub accounts?

### DISCUSSION THREAD

**Discussion by korosuke613** [github/gh-actions-importer#71] [Feedback] [answered]
I am trying to use the `audit` and `forecast` commands to migrate from CircleCI to GitHub Actions. I want to do this with my personal account, but when I run the commands, it runs the API for GitHub Organization and fails.

> (GET 404) Not Found: https://api.github.com/orgs/korosuke613/repos?per_page=100

```console
$ gh actions-importer audit circle-ci --output-dir tmp/audit
[2023-03-08 08:23:30] Logs: 'tmp/audit/log/valet-20230308-082330.log'
[2023-03-08 08:23:30] Auditing 'https://app.circleci.com/pipelines/github/korosuke613'
[2023-03-08 08:23:31] Resource not found                                        
                      (GET 404) Not Found: https://api.github.com/orgs/korosuke613/repos?per_page=100
Extracting all pipelines: |=---=---=---=---=---=---=---=---=---=---=---=---=---|
```

```console
$ gh actions-importer forecast circle-ci --output-dir tmp/forecast_reports --start-date 2022-09-02
[[2023-03-08 08:31:46] Logs: 'tmp/forecast_reports/log/valet-20230308-083146.log'
[2023-03-08 08:31:46] Forecasting 'https://app.circleci.com/pipelines/github/korosuke613'
[2023-03-08 08:31:47] Resource not found                                        
                      (GET 404) Not Found: https://api.github.com/orgs/korosuke613/repos?per_page=100
Extracting jobs: |=---=---=---=---=---=---=---=---=---=---=---=---=---=---=---=|
```

Do the `audit` and `forecast` commands support personal GitHub accounts? I would like to know how to configure it to run the Users API([GET `/users/{username}/repos`](https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-repositories-for-a-user)) instead of the Organization API.

**Comment by ethanis** [comment, marked as answer]
@korosuke613 unfortunately no, the `audit` and `forecast` commands are not supported for user accounts migrating from CircleCI. This is a known limitation that is documented [here](https://docs.github.com/en/actions/migrating-to-github-actions/automated-migrations/migrating-from-circleci-with-github-actions-importer#limitations).

**Comment by calebmazitama** [comment]
How to overcome the challenges when audit and forecast commands to migrate from CircleCI to GitHub Actions?
